### DISCUSSION TITLE
Freac Bugs

### DISCUSSION THREAD

**Discussion by vk5trm** [enzo1982/freac#277] [Help & Support] [answered]
Bug 1 When processing a large number of file in the joblist it doesn't seem to do all the files in the joblist even though they are all selected. It might miss about 10 files in 100 on the list. I have got around it by keeping the joblist and run the joblist twice to get the ones that it has missed

Bug 2 The tagging puts the data from the previous file in a field if the currant file hasn't got any data in it. It also keeps truncating the filename and name of the song in the id tag and filename,This totally messes up the music tag and you have to go though all the files to fix them up manually. It needs to clear all the fields of the tag before it moves onto the next file.

**Comment by enzo1982** [comment, marked as answer]
Please see my answer in #272
